src/miniflask/miniflask.py

```diff
@@ -184,40 +172,7 @@ def print_recently_loaded(self, prepend="", loading_text=highlight_loading):
 
     def import_module(self, module_name):
         module_spec = self.modules_avail[module_name]
-        importpath = module_spec["importpath"]
-
-        # system import (e.g. miniflask internal modules or pip-module repositories)
-        if importpath == "system":
-            return import_module(module_spec["id"])
-
-        try:
-            direct_import = import_module(module_spec["id"])
-            if direct_import:
-                return direct_import
-        except ModuleNotFoundError:
-            pass
-
-        # imports across filesystem
-        # the random id (_instance_id) ensures sys.modules does not cache different miniflask instances
-        # (first we need to load the parent module, if available)
-        parent_module_name, rest = "miniflask." + self._instance_id + "." + path.basename(path.abspath(module_spec["importpath"])), ""
-        spec = ImportPathFinder().find_spec(parent_module_name, [path.dirname(importpath)])
-        if spec is None:
-            if rest:
-                raise ValueError("Could not import parent Module named '%s'. This is needed for module named '%s' (defined in '%s')." % (parent_module_name, module_spec["id"], module_spec["importpath"]))
-            raise ValueError("Module named '%s' (defined in '%s') could not be imported." % (module_spec["id"], module_spec["importpath"]))
-        if spec.loader is None:
-            raise ValueError("Could not import parent Module named '%s'. This is needed for module named '%s' (defined in '%s'). Did you maybe miss to define a `__init__.py` file in any subfolder?" % (parent_module_name, module_spec["id"], module_spec["importpath"]))
-
-        # set parent module name to identifier given by dict (base_id)
-        parent_module_name = "miniflask." + self._instance_id + "." + module_spec["base_id"]
-        spec.loader.name = parent_module_name
-
-        # actually load the parent module
-        spec.loader.load_module()
-
-        # now load the requested module
-        return import_module(parent_module_name + "." + module_spec["importname"])
+        return import_module(module_spec["importname"])
```

Comment:
In a previous fix, we used variable `id` instead of `importname`. Is this still relevant here?

Reply:
In the example above,
- the `id` of `module2` is `otherdir.module2`, which miniflasks view of the module.
- `importname` however is pythons view, which contains the names of the parent-modules as well, namely `myproject.subpackage.modules.otherdir.module2`. This is the name one would use to import `module2` from any python file in the project.

The reason that we changed this here is that this module intents to unify the naming as much as possible in miniflask as well as in python.

I hope this clarifies things.